apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
Another option. In applab.js have `var ASSET_PATH_PREFIX = '/v3/assets'`.

Then expose `Applab.__TestInterface.replaceAssetPathPrefix(newPath)` or maybe tack it onto the existing global, i.e. `window.__TestInterface.replaceApplabAssetPath(newPath)`.

Finally, in runlevelTest.js, call `replaceAssetPath`, with a single constant if we can get away with it, or with our per test value if that's necessary. If it is per test, could you also add a member to the example object in levelTest.js along with a comment?